### ISSUE TITLE
Update Vaccination age range UI

### ISSUE THREAD

**Issue by CarlosNZ** [msupply-foundation/open-msupply#4928]
As suggested by Maria's designs, the UI for inputting Vaccination age ranges could do with some improvement. This is the preferred option:

![photo_2024-10-01 11 30 40](https://github.com/user-attachments/assets/5316f70c-14ec-4f9b-8aa5-6ee57c3d1473)

You add periods outside the table, in *either* months or years, and then **Add** them:

![photo_2024-10-01 11 34 11](https://github.com/user-attachments/assets/277cdfb5-2976-4032-93f6-a4fd510a4eed)

Another suggestion was if we kept the existing UI, to put an explanatory message above:
![photo_2024-10-01 11 30 44](https://github.com/user-attachments/assets/a5202972-5cc6-4fa0-9073-00cb7c3eb67f)

I *don't* think this is a good idea -- we shouldn't be relying on helpers to make up for unclear UI, especially if there's a clearer solution available.

The only downside of the preferred option is not being able to enter a range in both months and years, e.g. 1 year, 6 months. This would require some discussion, but it seems to me it's fine to think in "months" if you're dealing with partial years (i.e. it's common to say "18 months" for a child's age and then deal in whole years past a certain point, like 2 years old).

Needs further discussion / agreement...

**Comment by andreievg** [comment]
Triage, just a suggestion to check the scope, seeing as it's one time configuration and sounds like the requirement has already been met

**Comment by andreievg** [comment]
Also wasn wondering about Mariya's design time, in terms of priority, is this something that you see as quite urgent @mark-prins ?

**Comment by mark-prins** [comment]
not urgent - we can get by with the current ui. I think it needs changing though - the wall of input boxes is hard to parse visually!
and I don't think the loss of functionality we get when dropping year and month entry is a big deal, not enough to worry me. I'd go with the single input for number and a drop down for month/year.

**Comment by andreievg** [comment]
Another issue: https://github.com/msupply-foundation/open-msupply/issues/6102
